### PR TITLE
docs: add jsonSchema field

### PR DESCRIPTION
**Pull request by jannyHou**
## Description

Add document according to PR https://github.com/strongloop/loopback-next/commit/d0014c607eb130bf0d2adaf3249e67c3755771ae

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/Model.md [hunk 1]
```diff
@@ -215,6 +215,11 @@ The complete list of valid attributes for property definitions can be found in
 LoopBack 3's
 [Model definition section](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#properties).
 
+You can also specify validation rules in the field `jsonSchema`, check the
+documentation of
+[Parsing requests](https://loopback.io/doc/en/lb4/Parsing-requests.html#request-body)
```
Comment (raymondfeng) [inline docs/site/Model.md]: What about `[Parsing requests](Parsing-requests.md#request-body)`?

File: docs/site/Model.md [hunk 2]
```diff
@@ -215,6 +215,11 @@ The complete list of valid attributes for property definitions can be found in
 LoopBack 3's
 [Model definition section](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#properties).
 
+You can also specify validation rules in the field `jsonSchema`, check the
```
Comment (raymondfeng) [inline docs/site/Model.md]: check -> check out?

Comment (raymondfeng) [reply docs/site/Model.md]: Can you provide an example here?

Comment (raymondfeng) [reply docs/site/Model.md]: I realized you added an example to `Parsing requests`.

Comment (jannyHou) [reply docs/site/Model.md]: yeah I think it's better to have the example in both places.

File: docs/site/Parsing-requests.md [hunk 3]
```diff
@@ -179,6 +179,29 @@ in/by the `@requestBody` decorator. Please refer to the documentation on
...
+      minLength: 10,
+    },
+  })
+  public name: string;
+}
+```
+
+A full list of validation keywords could be found in the
+[documentation of AJV validation keywords](https://github.com/epoberezkin/ajv#validation-keywords).
+
```
Comment (bajtos) [inline docs/site/Parsing-requests.md]: The transition from "A full list of validation keywors" to "We support `json`, `urlencoded` and `text` content types" seems awkward to me. I am proposing to introduce a new subsection header here.

```markdown
A full list of validation keywords could be found in the
[documentation of AJV validation keywords](https://github.com/epoberezkin/ajv#validation-keywords).

### Content Types

We support `json`, `urlencoded`, and `text` content types. (etc.)
```

**Pull request merged**